packages/db/postgres/tables.sql

```diff
@@ -275,6 +296,8 @@ CREATE TABLE IF NOT EXISTS psa_pin_request
   updated_at      TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL
 );
 
+CREATE INDEX IF NOT EXISTS psa_pin_request_content_cid_idx ON psa_pin_request (content_cid);
```

Comment:
While this sounds like a sensible index to have, can you give the rationale why this was added as part of this PR?

Reply:
This was added for #1043 to improive perfomance when getting pinned storage amopunt. Seems to have been removed in a merge from main, so its being reintstated here.